p/irish-rail/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const products = require('./products')
+
+const transformReqBody = (ctx, body) => {
+	body.client = {
+		type: 'IPA',
+		id: 'IRISHRAIL',
+		v: '4000100',
+		name: 'IrishRailPROD-APPSTORE',
+		os: 'iOS 12.4.8',
+	}
+	body.ver = '1.18'
+
+	body.auth = {type: 'AID', aid: 'P9bplgVCGnozdgQE'}
+	body.lang = 'de'
```

Comment:
Is this intentional?

Reply:
Nope. This should probably be `en`, right?

Reply:
Yes, or `ga`, not sure.